db/controller.js

```diff
@@ -5,5 +5,13 @@ module.exports.getApi = function(req, res) {
 }
 
 module.exports.getApiTopics = function(req, res) {
-    res.status(200).send({topics: model.topics})
+  res.status(200).send({topics: model.topics})
+}
+
+module.exports.getArticleById = function(req, res) {
+  model.fetchArticleById(req.params.article_id)
+  .then(({rows}) => {
+    if (rows.length) res.status(200).send({article: rows[0]})
+    else res.status(404).send({msg: "No such article"})
+  })
```

Comment:
This works, but it is not how error handling should be done.

Have a read of error handling notes for this as well.